i18nify-data/currency/data.json

```diff
@@ -1182,50 +997,28 @@
       "numeric_code": "376",
       "minor_unit": "2",
       "symbol": "₪",
-      "physical_currency_denominations": [
-        "20",
-        "50",
-        "100",
-        "200"
-      ]
+      "physical_currency_denominations": ["20", "50", "100", "200"]
     },
     "JMD": {
       "name": "Jamaican Dollar",
       "numeric_code": "388",
       "minor_unit": "2",
-      "symbol": "$",
-      "physical_currency_denominations": [
-        "50",
-        "100",
-        "500",
-        "1000",
-        "5000"
-      ]
+      "symbol": "J$",
```

Comment:
Not verifying these symbols. Hope these are correct and as per our last discussion.

Reply:
Yes.

These have picked up directly from this sheet: https://docs.google.com/spreadsheets/d/13VFVLJql-IPeIV5NTR5MNzeeJp5LVkelbt4jFGUpBlE/edit#gid=0 that we created post our discussion. We have approvals from product here.